enterprise/migrations/0140_update_enrollment_sources.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+ENTERPRISE_ENROLLMENT_SOURCES = {
```

Comment:
nit: might rename to 'ADDITIONAL_ENTERPRISE_ENROLLMENT_SOURCES'

Reply:
If the dict has the keyword `additional` then perhaps also a change to the methods below as well? `add_new_enterprise_enrollment_source` or something similar. I haven't entirely convinced myself yet so feel free to push back.